Source/Core/DolphinQt/Config/InfoWidget.cpp

```diff
@@ -181,6 +194,52 @@ QGroupBox* InfoWidget::CreateBannerDetails()
   return group;
 }
 
+QGroupBox* InfoWidget::CreateTimePlayedDetails()
+{
+  const auto update_time_played = [this](const std::string& game_id,
+                                         const std::chrono::milliseconds time_played) {
+    if (game_id != m_game.GetGameID())
+      return;
+
+    using namespace std::chrono_literals;
+
+    const auto hours_played = time_played / 1h;
+    const auto minutes_played = (time_played % 60min) / 1min;
+
+    // i18n: A time displayed as hours and minutes
+    const QString time_played_string =
+        tr("Time Played: %1h %2m").arg(hours_played).arg(minutes_played);
+    m_time_played->setText(time_played_string);
+  };
+
+  m_time_played = new QLabel;
+  const std::chrono::milliseconds time_played =
+      TimePlayedManager::GetInstance().GetTimePlayed(m_game.GetGameID());
+  update_time_played(m_game.GetGameID(), time_played);
+
+  auto* const edit_time_played = new QPushButton(tr("Edit Time Played"));
+  connect(edit_time_played, &QPushButton::clicked,
+          [this]() { new EditTimePlayedDialog(this, m_game); });
+
+  auto* const layout = new QFormLayout;
+  layout->addRow(m_time_played, edit_time_played);
+  setSizePolicy(QSizePolicy::Preferred, QSizePolicy::Fixed);
```

Comment:
```suggestion
  auto* const layout = new QHBoxLayout;
  layout->addWidget(m_time_played, 1, Qt::AlignHCenter);
  layout->addWidget(edit_time_played);
```

The `setSizePolicy` call is messing up resizing in the Info tab, and there's unused space to the right of the button.

Reply:
Actually I can just add some stretch to the main `InfoWidget` layout.